Packs/AzureStorageTable/Integrations/AzureStorageTable/AzureStorageTable.yml

```diff
@@ -0,0 +1,278 @@
+      isArray: false
+      name: row_key
+      required: true
+      secret: false
+    - default: false
+      description: 'Entity records in JSON format: { "Key1": Value1, "Key2": Value2}'
+      isArray: false
+      name: entity_fields
+      required: true
+      secret: false
+    deprecated: false
+    description: Update an existing entity in a table.The Update Entity command does
+      not replace the existing entity.
+    execution: false
+    name: azure-storage-table-entity-update
+  - arguments:
+    - default: false
+      description: Table name.
+      isArray: false
+      name: table_name
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for the partition within a given table.
+      isArray: false
+      name: partition_key
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for an entity within a given partition.
+      isArray: false
+      name: row_key
+      required: true
+      secret: false
+    - default: false
+      description: 'Entity records in JSON format: { "Key1": Value1, "Key2": Value2}'
+      isArray: false
+      name: entity_fields
+      required: true
+      secret: false
+    deprecated: false
+    description: Replace an existing entity in a table.The Replace Entity command
+      replace the entire entity and can be used to remove properties.
+    execution: false
+    name: azure-storage-table-entity-replace
+  - arguments:
+    - default: false
+      description: Table name.
+      isArray: false
+      name: table_name
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for the partition within a given table.
+      isArray: false
+      name: partition_key
+      required: false
+      secret: false
+    - default: false
+      description: Unique identifier for an entity within a given partition.
+      isArray: false
+      name: row_key
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        Filter Entities query expression.
+        Information about Query expression structure can be found here: https://docs.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#constructing-filter-strings
+      isArray: false
+      name: filter
+      required: false
+      secret: false
+    - default: false
+      description: Comma-separated Entity properties to return.
+      isArray: false
+      name: select
+      required: false
+      secret: false
+    - default: true
+      defaultValue: '50'
+      description: |-
+        Number of entities to retrieve. Default is 50.
+        This argument is unusable when 'partition_key' is provided.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: true
+      defaultValue: '1'
+      description: |-
+        Page number. Default is 1.
```

Comment:
What about adding to both limit and page that one cannot be used without the other, if that's the case?
And verifying it in the code as well?

Reply:
Actually, because there are default values for these arguments, the user can define only one of them (so that the value of the second argument will be equal to the default value).
Therefore in my opinion it is irrelevant in this case.